### DISCUSSION TITLE
Hand controlled LED with Raspberry Pi Pico instead of AMB23

### DISCUSSION THREAD

**Discussion by bidrohini** [micropython/micropython#10953] [General]
Hello, I am planning to start doing some elementary machine-learning projects. I was searching.  Then this project caught my eyes. 

https://www.pcbway.com/project/shareproject/Hand_Controlled_LED_OpenCV_MicroPython_IoT_a96b1a01.html

This is a hand-controlled LED. This is done with AMB23. The problem is,  I do not have an AMB23. I have a Raspberry Pi Pico. My question is, is it possible to do this project with Raspberry Pi Pico?

**Comment by peterhinch** [comment]
The MicroPython part of this project is very simple and can readily be adapted for other targets. However the system is dependent on a network connection to the PC so you'd need a Pico W or similar.

**Reply by bidrohini** [reply]
Thank you, sir. This is something I must have noticed earlier. But actually I do not have the W version. I have the ordinary Raspberry Pi Pico. If I add a wifi module with it (like the ESP8266) will it do?

**Reply by peterhinch** [reply]
Just use the ESP8266 and keep the Pico for something else.
